int32 command
float64 x_feedback
float64 x_target
float64 y_feedback
float64 y_target
float64 z_feedback
float64 z_target 
float64 yaw_feedback
float64 yaw_target
float64 data

rmg2/rmg2_pid xpid
rmg2/rmg2_pid ypid
rmg2/rmg2_pid zpid
rmg2/rmg2_pid yawpid
rmg2/rmg2_pid qxpid
rmg2/rmg2_pid qypid
